src/Pages/Income.cshtml

```diff
@@ -93,7 +93,7 @@
                         </td>
                         <td class="border-primary">
                             <a asp-page="/EditEntry" asp-route-id="@item.Id" asp-route-entryType="1" class="btn btn-sm btn-success">Edit </a>
-                            <input asp-page-handler="Delete" asp-route-id="@item.Id" type="submit" class="btn btn-sm btn-danger" value="Delete" />
+                            <input asp-page-handler="Delete" asp-route-id="@item.Id" asp-route-userId="@item.UserId"  type="submit" class="btn btn-sm btn-danger" value="Delete" />
```

Comment:
Why was this added? Same in expenses.cshtml

Reply:
Beforehand we had I believe goals requiring both id and user id to delete, so I made it standard that to delete something from the database you need to pass both the user id and the id.
This was added to the front end to pass this the user id as a variable to the Delete method, but maybe there's another way to do this without adding code to the front end?